KyberElasticAdapter/contracts/interfaces/IERC20.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IERC20 {
+    event Approval(address, address, uint256);
+    event Transfer(address, address, uint256);
+
+    function name() external view returns (string memory);
+
+    function decimals() external view returns (uint8);
+
+    function transferFrom(
+        address,
+        address,
+        uint256
+    ) external returns (bool);
+
+    function allowance(address, address) external view returns (uint256);
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.html